openedx/core/lib/token_utils.py

```diff
@@ -98,3 +98,12 @@ def encode(self, payload):
             algorithm = self.jwt_auth['JWT_ALGORITHM']
 
         return jwt.encode(payload, secret, algorithm=algorithm)
+
+
+def time_since_epoch():
```

Comment:
Thanks for adding this utility function - however, there's only caller to this function.
I was actually thinking instead about adding a common place where we define the expiration time for DOT-issued `RestrictedApplication` tokens.  This way, if we ever choose to change that value, it would be in one place.  And also, it isn't hard-coded in multiple places in the code.